src/js/pages/GranthIndex/raagIndices.js

```diff
@@ -221,7 +221,43 @@ export default {
       },
       {
         name: 'Raag Jaijavanti',
-        pages: [1352, 1353],
+        pages: [1352, 1352],
+      },
+      {
+        name: 'Salok Sehaskriti Mahalla 1',
+        pages: [1353, 1353],
+      },
+      {
+        name: 'Salok Sehaskriti Mahalla 5',
```

Comment:
@tarunsingh5 I remember some discussion about the transliteration for Sehaskriti. Any conclusion?